README.md

```diff
@@ -25,6 +25,10 @@ After you install the module, you can use this theme in a build by running:
 
 Build the plugin by running `yarn build:plugin`.
 
+After you install the module, you can use this plugin in a build by running:
+
+`typedoc --theme ./node_modules/typedoc-neo-theme/bin/default --plugin typedoc-neo-theme`
```

Comment:
Alternatively it can be [partially placed in the typedoc options](https://github.com/actions-on-google/actions-on-google-nodejs/blob/master/package.json#L23): `typedoc --options typedoc.json --theme node_modules/typedoc-neo-theme/bin/default`

Reply:
updated